deepmd/pd/model/descriptor/dpa3.py

```
@@ -0,0 +1,574 @@
+@BaseDescriptor.register("dpa3")
+class DescrptDPA3(BaseDescriptor, paddle.nn.Layer):
+    def __init__(
+        self,
+        ntypes: int,
+        # args for repflow
+        repflow: Union[RepFlowArgs, dict],
+        # kwargs for descriptor
+        concat_output_tebd: bool = False,
+        activation_function: str = "silu",
+        precision: str = "float64",
+        exclude_types: list[tuple[int, int]] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)